runtime/autoload/provider/python.vim

```diff
@@ -1,19 +1,22 @@
-" The python provider uses a python host to emulate an environment for running
-" python-vim plugins(:h python-vim). See :h nvim-providers for more
-" information.
+" The Python provider uses a Python host to emulate an environment for running
+" python-vim plugins. See ":h nvim-provider" for more information.
```

Comment:
All other instances of this use `:help` instead or `:h`, so it should be probably be done here.